File: UniversityApiBackend/Models/DataModels/Address.cs
namespace UniversityApiBackend.Models.DataModels
{
    public class Address
    {
        public int Id { get; set; }
        public string? Street { get; set; }
        public string? City { get; set; }
        public string? State { get; set; }
        public string? ZipCode { get; set; }
        public string? Country { get; set; }
     public string? Comunity { get; set; }
        public ICollection<Student> Students { get; set; } = new List<Student>();
    }
}

File: Frontend/front-university/src/app/services/api/models/address.ts
/* tslint:disable */
/* eslint-disable */
import { Student } from './student';
export interface Address {
  city?: null | string;
  comunity?: null | string;
  country?: null | string;
  id?: number;
  state?: null | string;
  street?: null | string;
  students?: null | Array<Student>;
  zipCode?: null | string;
}
